content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+
+h3. Example
+
+For example, to specify the @rewind@ channel param with the value @"1"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '1'}}
+  const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+To modify the @rewind@ channel param with the value @"15s"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '15s'}}
+  channel.setOptions(channelOpts, (err) => {
+    if(!err) {
+      console.log('channel params updated')
+    }
+  });
+```
+
+h2(#using-parameters-with-other-transports). Using parameters with non-Ably transports
+
+It is possible to interact with Ably channels using transports that do not involve using an Ably library; for example using SSE without any library, or using a supported non-Ably protocol such as MQTT. In these cases, it is also necessary to use a qualified channel name.
+
+A set of params is expressed by including a query string, using standard URL query syntax and encoding, within the qualifier part of a channel name. The qualifier part is in square brackets at the start of the channel name.
+
+For example, to specify the parameter @foo@ with value @bar@ on channel @baz@, the qualified channel name would be @[?foo=bar]baz@. If the channel name already has a qualifier, such as @[meta]log@, then the query string follows the existing qualifier, as in @[meta?foo=bar]log@.
+
+In an SSE connection, it is also possible to specify channel parameters as a query string in the connection URL, instead of as a qualifier on an individual channel name. In this case, the given channel parameters apply to all channel attachments associated with that connection.
+
+h3. SSE example
+
+For example, to specify the @rewind@ channel param with the value @"1"@ using a querystring parameter, where it will apply to all channels:
+
+```[javascript]
+var querystring = 'v=1.2&channels={{RANDOM_CHANNEL_NAME}}&rewind=1&key={{API_KEY}}';
+var eventSource = new EventSource('https://realtime.ably.io/event-stream?' + querystring);
+```
+
+Or to specify the same parameter but only applying to one channel of two, using a qualified channel name:
+
+```[javascript]
+  var channelOne = encodeURIComponent('[?rewind=1]channel1');
+  var channelTwo = 'channel2';
+  var channels = channelOne + ',' + channelTwo;
+  var querystring = 'v=1.2&key={{API_KEY}}&channels=' + channels';
+  var eventSource = new EventSource('https://realtime.ably.io/event-stream?' + querystring);
+```
+h2(#next-steps). Next steps
```

Comment:
I think this documentation should include an API reference

Reply:
@tomczoink re api reference